app/views/admin/collections/show.html.erb

```diff
@@ -30,21 +50,26 @@ Unless required by applicable law or agreed to in writing, software distributed
     </div>
     <div class="col-md-3">
       <p><strong>Poster:</strong></p>
-      <p><%= @collection.poster.original_name if @collection.poster.present? %></p>
-      <% if @collection.poster.present? %>
+      <p id="poster_original_name"><%= @collection.poster.original_name if @collection.poster.present? %></p>
       <div class="well-vertical-spacer">
-        <%= image_tag(poster_admin_collection_path(@collection), class: 'img-responsive') %>
+        <% if @collection.poster.present? %>
+          <%= image_tag(poster_admin_collection_path(@collection), class: 'img-responsive', id: 'poster-image') %>
+        <% else %>
+          <img src="" class="img-responsive" id="poster-image">
```

Comment:
Does this need to be in there with an empty `src` tag?

Reply:
Yeah, the cropper js requires this and adds a src value.